cpp/include/cudf/io/parquet_metadata.hpp

```diff
@@ -21,14 +21,16 @@
 
 #pragma once
 
+#include <cudf/io/parquet_schema.hpp>
 #include <cudf/io/types.hpp>
 #include <cudf/utilities/export.hpp>
 
-#include <optional>
 #include <string_view>
-#include <variant>
 #include <vector>
 
+//! Parquet physical `Type`
+using cudf::io::parquet::Type;
```

Comment:
should this be in the cudf::io namespace?

Reply:
I mean should the `using` here be in the namespace?

Reply:
I see. yeah of course

Reply:
All done!